pkg/costattribution/active_tracker.go

```diff
@@ -16,10 +16,18 @@ import (
 	"go.uber.org/atomic"
 )
 
+type asCounters struct {
+	asCount     *atomic.Int64
+	nhCount     *atomic.Int64
+	nhBucketNum *atomic.Int64
+}
```

Comment:
Opinionated, but given it's `asCounters`, adding `Count` suffix to the field is redundant and doesn't help readability. 

OTOH, do we need `as` prefix in the struct? Isn't that implicit from this entire feature?

IMO this reads better:
```suggestion
type counters struct {
	activeSeries           *atomic.Int64
	nativeHistograms       *atomic.Int64
	nativeHistogramBuckets *atomic.Int64
}
```